app/assets/stylesheets/active_admin/components/_table_tools.scss

```diff
@@ -1,19 +1,22 @@
 .table_tools {
-  @include clearfix;
   margin-bottom: 16px;
+  @include clearfix;
 }
 
 .table_tools .dropdown_menu {
   float: left;
 }
 
 a.table_tools_button, .table_tools .dropdown_menu_button {
-  @include light-button;
   @include gradient(#FFFFFF, #F0F0F0);
   @include border-colors(#d9d9d9, #d0d0d0, #c5c5c5);
-  font-size: 0.9em;
-  padding: 4px 14px 4px;
-  margin: 0;
+  @include light-button;
```

Comment:
Wouldn't this change be different visually too since this does use the gradient/border-colors mixing locally to override the values from the button mixin? The same would apply in some other areas since they have the same type of change.

Reply:
I see your point. Your logic is correct but I didn't notice changes in my manual check. 
I'll double-check this and come back.

Reply:
@javierjulio You were right, I was looking at the wrong component. Sorry! this should be the last change.